tailnet/coordinator.go

```diff
@@ -40,7 +40,10 @@ func ServeCoordinator(conn net.Conn, updateNodes func(node []*Node) error) (func
 			}
 			err = updateNodes(nodes)
 			if err != nil {
-				errChan <- xerrors.Errorf("update nodes: %w", err)
+				select {
```

Comment:
Perhaps we should make the `errChan` only of size 1 (to make the behavior/expectation explicit) and maybe create a function called `sendErr()` that does the `select`. Right now I suppose these errors could fill up the `errChan` and then the `Decode` above would still hang?

Reply:
Good idea!